### ISSUE TITLE
Web - Tagtree - Tagset icon

### ISSUE THREAD

**Issue by gusferguson** [openmicroscopy/design#26]
@will-moore @pwalczysko 

This is the tags icon with slightly different colours.

Question is - would it be better to go radical and have totally different colours as in Project-Dataset.

I would have to completely redo the tags and tag icons anyway - so a radical colour change is not any more work.

1. 10 degree colour rotation

![test tagset icon 10 degrees](https://cloud.githubusercontent.com/assets/1552703/13772398/d1a68d60-ea8b-11e5-95cb-2bd81fb0bb76.png)

2. 15 degree colour rotation

![test tagset icon 15 degrees](https://cloud.githubusercontent.com/assets/1552703/13772404/d80396bc-ea8b-11e5-8d5e-37135de8e4dc.png)

**Comment by pwalczysko** [comment]
yes, I can see the tag from tagset much more readily now. Completely different color would not be bad either.

**Comment by gusferguson** [comment]
@pwalczysko @will-moore 

I have had to regenerate the tag and tags icon to do the mockup, so introduced a bit more definition in them.

Below shows:
- a purple/dark grey tags icon in the toolbar - based on the data tree screen icon colours
- a same grey colour tags icon at the top of the tag tree - just a bit more definition
- a green tags icon - based on the dataset icon colours

Top most tag icon is new one with more definition.

What are your thoughts?

![new tagset icons 2](https://cloud.githubusercontent.com/assets/1552703/13862951/5da6a346-ec8e-11e5-9c08-a8ee9dca498b.png)

**Comment by pwalczysko** [comment]
I like the dark grey/purple color for the tagset. (green resembles too much the Dataset, and grey (the same color as tags) does not do it for me.

**Comment by gusferguson** [comment]
I think I prefer the green and I think the dataset-tagset association is good.

@will-moore - casting vote/veto!

**Comment by will-moore** [comment]
I think I prefer the 15 degree colour change (or even the grey) but I'm not sure the Dataset association is good because Tagset (green) and Tag (blue) then look similar to Project (blue) and Dataset (green) except that the hierarchy is the other way up. And I also think that it's nice to have a clearer distinction anyway - especially since they can all be jumbled up together in the Tag tree.
 
![screen shot 2016-03-18 at 10 04 39](https://cloud.githubusercontent.com/assets/900055/13874634/18610980-ecf1-11e5-849d-e6ab1479ef14.png)

**Comment by gusferguson** [comment]
The colour rotation is certainly easier to do - I just use the existing icon.

Here are the icons for 10, 12, 15 and 20 degree rotations - see what you think when they are in place.

10 degree colour rotation:

![left_sidebar_icon_tags 10 degree](https://cloud.githubusercontent.com/assets/1552703/13875483/084750ea-ecf6-11e5-8e81-982d5a99316d.png)

12 degree colour rotation:

![left_sidebar_icon_tags 12 degree](https://cloud.githubusercontent.com/assets/1552703/13875485/0882506e-ecf6-11e5-9108-115b642e0015.png)

15 degree colour rotation:

![left_sidebar_icon_tags 15 degrees](https://cloud.githubusercontent.com/assets/1552703/13875484/087b09b2-ecf6-11e5-9b87-687fe582f98d.png)

20 degree colour rotation:

![left_sidebar_icon_tags 20 degrees](https://cloud.githubusercontent.com/assets/1552703/13875534/4c7165a8-ecf6-11e5-9113-699546d511d1.png)


(Don't look a lot different in isolation)

**Comment by hflynn** [comment]
Not to throw a spanner in the works, but while you're redesigning this anyway, have you considered accessibility issues regarding colour and contrast? There seem to be lots of online tools to help with this stuff.

**Comment by gusferguson** [comment]
@hflynn - I did a full review of all the icons and text with respect to contrast and colour a year or so ago, and it passed with at least an AA according to the W3C WCAG 2.0 rules. I use a desktop tool Colour Contrast Analyser.app https://www.paciellogroup.com/resources/contrastanalyser/.

I do keep an eye on it with changes, but if you detect any issues please shout.

**Comment by hflynn** [comment]
That's great @gusferguson - I defer to your expertise, it just occurred to me that distinguishing green from grey here might be an issue so I wanted to check such things were being kept in mind.

**Comment by gusferguson** [comment]
@hflynn - the colour issue as such is a whole different can of worms - we have never looked at how any of our UIs and web pages work with colour blindness, or for that matter low visual acuity. In this instance sticking to shades of grey is probably safest, but if there are purple shades (red) in there the contrast for Red-green colour blindness could be affected. In this case the contrast does not fall off too badly for the 15 and 20 degree rotations - which is moving towards red.

15 degree contrast:

![colour contrast analyser](https://cloud.githubusercontent.com/assets/1552703/13876517/c9c31b9a-ecfc-11e5-85a9-7befc72fb89f.png)
